app/cdap/api/replicator.js

```diff
@@ -51,6 +51,7 @@ export const MyReplicatorApi = {
   assessPipeline: apiCreator(dataSrc, 'POST', 'REQUEST', `${draftPath}/assessPipeline`),
   validatePipeline: apiCreator(dataSrc, 'POST', 'REQUEST', validatePipelinePath),
   assessTable: apiCreator(dataSrc, 'POST', 'REQUEST', `${draftPath}/assessTable`),
+  assessTableNoDraft: apiCreator(dataSrc, 'POST', 'REQUEST', `${servicePath}/assessTable`),
```

Comment:
Lets get a better name for this - maybe describe table? Thats what this is doing right? describing what the table target values would be?